phpBB/download/file.php

```diff
@@ -11,310 +11,45 @@
 *
 */
 
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
 /**
 * @ignore
 */
 define('IN_PHPBB', true);
 $phpbb_root_path = (defined('PHPBB_ROOT_PATH')) ? PHPBB_ROOT_PATH : './../';
 $phpEx = substr(strrchr(__FILE__, '.'), 1);
+include($phpbb_root_path . 'common.' . $phpEx);
 
-// Thank you sun.
-if (isset($_SERVER['CONTENT_TYPE']))
-{
-	if ($_SERVER['CONTENT_TYPE'] === 'application/x-java-archive')
-	{
-		exit;
-	}
-}
-else if (isset($_SERVER['HTTP_USER_AGENT']) && strpos($_SERVER['HTTP_USER_AGENT'], 'Java') !== false)
-{
-	exit;
-}
+// Start session management
+$user->session_begin();
+$auth->acl($user->data);
+
+/** @var \phpbb\controller\helper $controller_helper */
+$controller_helper = $phpbb_container->get('controller.helper');
 
 if (isset($_GET['avatar']))
```

Comment:
Just use `$request->is_set('avatar')`?

Reply:
That is from the original file.php, i think that don't matter much, this file will have to be removed in the future